test.js

```diff
@@ -22,6 +22,11 @@ test('filnamify()', t => {
 	t.is(filenamify('con', {replacement: '🐴🐴'}), 'con🐴🐴');
 	t.is(filenamify('c/n', {replacement: 'o'}), 'cono');
 	t.is(filenamify('c/n', {replacement: 'con'}), 'cconn');
+	t.is(filenamify('!!!!!'), '!!!!!');
+	t.is(filenamify('/!!///!!!!/////'), '!!!!!!!');
+	t.is(filenamify('!!!!!//', {replacement: '!!'}), '!!!!!');
+	t.is(filenamify('foofoo//', {replacement: 'foo'}), 'foofoo');
+	t.is(filenamify('foofoo/foo/', {replacement: 'foo'}), 'foofoofoofoo');
```

Comment:
These assertions should ho in a new test to make it clear what they're testing. Also link to your PR in a comment if for future reference to the explanation of the assertions.

Reply:
I would also like to see some more assertions. While you say your fix is simple, you do introduce a lot of new code, which has the potential to break stuff in subtle ways.